daemon/daemon_main.go

```diff
@@ -376,8 +376,12 @@ func init() {
 	flags.String(option.ConfigDir, "", `Configuration directory that contains a file for each option`)
 	option.BindEnv(option.ConfigDir)
 
-	flags.Uint(option.ConntrackGarbageCollectorInterval, 60, "Garbage collection interval for the connection tracking table (in seconds)")
-	option.BindEnv(option.ConntrackGarbageCollectorInterval)
+	flags.Uint(option.ConntrackGarbageCollectorIntervalDeprecated, 0, "Garbage collection interval for the connection tracking table (in seconds)")
+	flags.MarkHidden(option.ConntrackGarbageCollectorIntervalDeprecated)
+	option.BindEnv(option.ConntrackGarbageCollectorIntervalDeprecated)
+
+	flags.Duration(option.ConntrackGCInterval, time.Duration(0), "Overwrite the connection-tracking garbage collection interval")
```

Comment:
Is 0 the default? I thought it was 12h as stated in the documentation.

Reply:
It's not the default, it makes `IsSet()` return false. The defaults are set in `pkg/defaults`